Description: Revert "ArmVirtPkg: make EFI_LOADER_DATA non-executable"
 The versions of GRUB most distros are shipping still depend on executable
 EFI_LOADER_DATA. Revert this upstream change until the necessary fixes are
 more generally available.
Author: dann frazier <dannf@debian.org>
Bug-Debian: https://bugs.debian.org/1025656
Forwarded: https://edk2.groups.io/g/devel/message/97814
Last-Update: 2023-03-09

--- a/ArmVirtPkg/ArmVirt.dsc.inc
+++ b/ArmVirtPkg/ArmVirt.dsc.inc
@@ -361,7 +361,7 @@
   # reserved ones, with the exception of LoaderData regions, of which OS loaders
   # (i.e., GRUB) may assume that its contents are executable.
   #
-  gEfiMdeModulePkgTokenSpaceGuid.PcdDxeNxMemoryProtectionPolicy|0xC000000000007FD5
+  gEfiMdeModulePkgTokenSpaceGuid.PcdDxeNxMemoryProtectionPolicy|0xC000000000007FD1
 
 [Components.common]
   #
